src/webportal/src/app/job/job-view/fabric/job-detail.jsx

```diff
@@ -40,7 +43,10 @@ import {
   NotFoundError,
   fetchRawJobConfig,
 } from './job-detail/conn';
-import { getHumanizedJobStateString } from '../../../components/util/job';
+import Card from './job-detail/components/card';
+import HorizontalLine from '../../../components/horizontal-line';
+import StatusBadge from '../../../components/status-badge';
+import TaskRoleContainerList from './job-detail/components/task-role-container-list';
 
 class JobDetail extends React.Component {
   constructor(props) {
```

Comment:
Seems job's task running, waiting, failed counters is missing

Reply:
Because the code is rewritten, is this feature important to have?